includes/license/notice.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace tangible\updater;
+use tangible\framework;
+use tangible\updater;
+
+$updater = updater::$instance;
+
+$version = $updater->version;
+$slug    = $plugin->name;
+$license = get_license_key($plugin);
+$license_status =get_license_status($plugin);
+
+$plugin->plugin_row_enqueued = [];
```

Comment:
I wondered how the `$plugin` instance can be used at the top level like this, and then saw this `notice.php` is included from within `register_plugin()` so all the local variables in the function closure are available. Nice solution!